pkg/kv/kvserver/allocator/plan/replicate.go

```diff
@@ -263,7 +263,7 @@ func (rp ReplicaPlanner) PlanOneChange(
 	// NB: the replication layer ensures that the below operations don't cause
 	// unavailability; see kvserver.execChangeReplicasTxn.
 	action, allocatorPrio := rp.allocator.ComputeAction(ctx, rp.storePool, conf, desc)
-	log.KvDistribution.VEventf(ctx, 1, "next replica action: %s", action)
+	log.KvDistribution.VInfof(ctx, 1, "next replica action: %s", action)
```

Comment:
This log line is needed to pass this test.

https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_BazelEssentialCi/11570626?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true&expandBuildProblemsSection=true&expandBuildTestsSection=true